src/migrations/mapping_api_version_0_0_2.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs-extra')
+const semver = require('semver')
+const toolbox = require('gluegun/toolbox')
+const yaml = require('js-yaml')
+const getGraphTsVersion = require('./util/versions').getGraphTsVersion
+
+// If any of the manifest apiVersions are 0.0.2, replace them with 0.0.3
+module.exports = {
+  name: 'Bump mapping apiVersion from 0.0.2 to 0.0.3',
+  predicate: async ({ sourceDir, manifestFile }) => {
+    // Obtain the graph-ts version, if possible
+    let graphTsVersion
+    try {
+      graphTsVersion = getGraphTsVersion(sourceDir)
+    } catch (_) {
+      // If we cannot obtain the version, return a hint that the graph-ts
+      // hasn't been installed yet
+      return 'graph-ts dependency not installed yet'
+    }
+
+    let manifest = yaml.safeLoad(fs.readFileSync(manifestFile, 'utf-8'))
+    return (
+      // Only migrate if the graph-ts version is > 0.5.1...
+      semver.gt(graphTsVersion, '0.5.1') &&
```

Comment:
The current version is 0.12.0, so this should be `0.12.0`, so it only takes effect starting with the next graph-ts release we make.